src/@dvcorg/gatsby-theme-iterative/data/styles.module.css

```diff
@@ -1,21 +1,21 @@
 .cmlIcon {
-  background-image: url("../../../../static/img/cml_icon-color--square_vector.svg");
+  background-image: url('../../../../static/img/cml_icon-color--square_vector.svg');
```

Comment:
`prettier` bleeds through our single-quote preference into CSS. Just a symptom of actually running prettier on CSS files again.